applications/app/controllers/CrosswordsController.scala

```diff
@@ -313,15 +313,19 @@ class CrosswordEditionsController(
     getCrosswords
       .map(parseCrosswords)
       .flatMap { crosswords =>
-        remoteRenderer.getEditionsCrossword(wsClient, crosswords)
+        crosswords.headOption match {
+          case Some(crossword) => remoteRenderer.getEditionsCrossword(wsClient, crossword)
```

Comment:
I'm not sure I've understood this change. It looks like this means the crosswords page now only includes a single crossword? Am I missing something? 🤔